src/components/ClickToCopySection/styles/ClickToCopySection.styled.ts

```diff
@@ -1,18 +1,18 @@
 import styled from 'styled-components'
 
 export const ClickToCopyTitle = styled.p`
-  margin: 0px;
+  margin: 0;
 
   font-style: normal;
-  font-weight: 500;
-  font-size: 18px;
+  font-weight: 300;
+  font-size: 1.25vw;
```

Comment:
would this make text too big if window width is too large? consider using clamp instead